docs/applications/simple-bank-account/README.md

```diff
@@ -128,12 +132,13 @@ scalar.dl.client.private_key_path=conf/client-key.pem
 If everything is set up properly you should be able to register your certificate on the ScalarDL network as
 
 ```bash
-$ ${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ./conf/client.properties
+$ cd contract
+$ ${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ./contract/conf/client.properties
```

Comment:
```suggestion
$ ${SCALAR_SDK_HOME}/bin/scalardl register-cert --properties ./contract/conf/client.properties
```
When I ran this command, I faced the following error, I think we need to update the file path.
```console
$ ${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ./contract/conf/client.properties
-bash: ../scalardl-java-client-sdk-3.9.1/client/bin/scalardl: No such file or directory
```

Reply:
I think it is same as https://github.com/scalar-labs/scalardl/pull/61#discussion_r1687597593.